05-UI-ReactJS/source/js/movieForm.js

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import {Movie} from './movie.js';
+
+class MovieForm extends React.Component {
+  constructor(props) {
+    super(props);
+    this.handleSubmit = this.handleSubmit.bind(this);
+    this.handleReset = this.handleReset.bind(this);
+    this.state = {
+      movie: new Movie()
+    };
+  }
+
+  render() {
+    let mode;
+    if(this.state.movie.id) {
+      mode = "editing";
+    } else {
+      mode = "creating";
+    }
+    return (
+      <div>
+        <h3>Movie</h3>
+        <h5 id="mode">You are {mode} a movie.</h5>
+        <form onSubmit={this.handleSubmit}>
+          <label htmlFor="title" id="movie-title">
+            Title:
+            <input onChange={this.handleChange("title")} value={this.state.movie.title} id="movie-title-input" />
```

Comment:
Are you sure you are calling this when component mount? I think this should be `this.handleChange.bind(this, "title")`